src/bignum.nr

```diff
@@ -6,7 +6,8 @@ use crate::fns::{
     constrained_ops::{
         add, assert_is_not_equal, conditional_select, derive_from_seed, div, eq, mul, neg, sub,
         udiv, udiv_mod, umod, validate_in_field, validate_in_range,
-    }, expressions::{__compute_quadratic_expression, evaluate_quadratic_expression},
+    },
+    expressions::{__compute_quadratic_expression, evaluate_quadratic_expression},
```

Comment:
Changes in any of these files aside `src/fns/unconstrained_helpers.nr` is simply from `nargo fmt`